ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/components/video-view-base/video-view-base.component.ts

```diff
@@ -24,7 +24,7 @@ export class VideoViewBaseComponent implements OnInit {
     this.disabledReplay = false;
   }
 
-  async replay() {
-    await this.videoViewComponent.play();
+  replay() {
```

Comment:
You removed async... ?

Reply:
Just testing for now. I don't have Safari, once this is built I'll test it